modules/extensions-support/src/main/resources/META-INF/services/org.mule.extension.introspection.declaration.spi.ModelEnricher

```diff
@@ -0,0 +1 @@
+org.mule.module.extension.internal.runtime.connector.ConnectorModelEnricher
```

Comment:
Agreed. Need to think about best way to do that.  IMO the "Connector Framework" needs to own this ordering extending from anything defined by the generic interceptor framework.